earthaccess/api.py

```diff
@@ -12,6 +12,12 @@
 from .utils import _validation as validate
 
 
+def _normalize_location(location: Union[str, None]) -> Union[str, None]:
```

Comment:
Total nit that shouldn't be taken seriously: does `Optional[str]` seem like a readability improvement at all?

I'm writing Typescript lately where you can do `location?: string` and that's pretty slick.

Reply:
(Preface: I'm not very steeped in type annotations) To me `Union` seems more appropriate here because this is a required positional argument, not a keyword with a default of `None`. But I don't care strongly either way as long as linting checks pass.

Reply:
>`Union` seems more appropriate here because this is a required positional argument, not a keyword with a default of `None`

Do you mean you think `Union` is more clear for that reason? Functionally they are the same thing, the argument would still be required: https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
> Do you mean you think Union is more clear for that reason?

Yes. Though I don't feel very strongly either way (or in general about type annotations). Happy to update if you'd prefer.

Reply:
I see the unclearness of `Optional` in that regard, good point! I also don't feel strongly, so let's leave it? :)